# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: security/sandbox/chromium/sandbox/win/src/policy_params.h
# Commit: 370c40e49230
# Full Hash: 370c40e4923073a87845b57c1f4851bacd9663da
# Author: Bob Owen <bobowencode@gmail.com>
# Date: 2019-06-12 21:42:35
# Regressor Bug: 1552160
# File Overlap Count: 3
# Description:
#   Bug 1552160 Part 1: Roll-up of chromium sandbox update and mozilla patches to get a running browser. r=jld,aklotz,tjr,bobowen
#   
#   This updates security/sandbox/chromium/ files to chromium commit 84108231f6e6e0772fb9a4643679ce76aa771e67.
#   
#   Existing and new patches applied from security/sandbox/chromium-shim/patches/with_update/ to give a compiling and mostly working browser.
# ==============================================================================

diff -r bd5ecd72f76c -r 370c40e49230 security/sandbox/chromium/sandbox/win/src/policy_params.h
--- a/security/sandbox/chromium/sandbox/win/src/policy_params.h	Wed Jun 12 21:55:59 2019 +1200
+++ b/security/sandbox/chromium/sandbox/win/src/policy_params.h	Wed Jun 12 11:10:48 2019 +0100
@@ -21,7 +21,7 @@
 // Policy parameters for file open / create.
 POLPARAMS_BEGIN(OpenFile)
   POLPARAM(NAME)
-  POLPARAM(BROKER)   // TRUE if called from the broker.
+  POLPARAM(BROKER)  // true if called from the broker.
   POLPARAM(ACCESS)
   POLPARAM(DISPOSITION)
   POLPARAM(OPTIONS)
@@ -30,7 +30,7 @@
 // Policy parameter for name-based policies.
 POLPARAMS_BEGIN(FileName)
   POLPARAM(NAME)
-  POLPARAM(BROKER)   // TRUE if called from the broker.
+  POLPARAM(BROKER)  // true if called from the broker.
 POLPARAMS_END(FileName)
 
 static_assert(OpenFile::NAME == static_cast<int>(FileName::NAME),
@@ -61,7 +61,6 @@
   POLPARAM(TARGET)
 POLPARAMS_END(HandleTarget)
 
-
 }  // namespace sandbox
 
 #endif  // SANDBOX_SRC_POLICY_PARAMS_H__